src/coin_test/orchestration/orchestration.py

```diff
@@ -231,25 +251,45 @@ def run(
             not saved to disk.
         slippage_calculator: Slippage calc to use in simulations. Defaults to Constant
         tx_calculator: Transaction calc to use in simulations. Defaults to Constant_tx
+        build_from_saved_results: Filepath to load saved results from.
 
     Returns:
         List: All results of backtests
+
+    Raises:
+        ValueError: If build_from_save_results isn't provided,
+            args must be provided to run a backtest
     """
-    if slippage_calculator is None:
-        slippage_calculator = ConstantSlippage()
-    if tx_calculator is None:
-        tx_calculator = ConstantTransactionFeeCalculator()
-
-    results = _gen_results(
-        all_datasets,
-        all_strategies,
-        slippage_calculator,
-        tx_calculator,
-        starting_portfolio,
-        backtest_length,
-        n_parallel,
-        output_folder,
-    )
+    if build_from_saved_results is not None:
+        results = _load(build_from_saved_results)
+
+    else:
+        if (
+            all_datasets is None
+            or all_strategies is None
+            or starting_portfolio is None
+            or backtest_length is None
+        ):
+            raise ValueError(
+                "Must specify 'build_from_save_results' or all of: 'all_datasets',\
+                    all_strategies', 'starting_portfolio', and 'backtest_length'"
+            )
+
+        if slippage_calculator is None:
+            slippage_calculator = ConstantSlippage()
```

Comment:
Maybe a log.info here stating No slippage defined, using Constant Slippage. Same for the transaction fees. Also for what it's worth, maybe we should default to no slippage or transaction fees?